lib/util/insightspublisher/index.js

```diff
@@ -60,6 +65,12 @@ class InsightsPublisher extends EventEmitter {
       },
       _WebSocket: {
         value: options.WebSocket
+      },
+      _isIpad: {
+        value: options.isIpad()
+      },
+      _isIphone: {
+        value: options.isIphone()
```

Comment:
@PikaJoyce I think you missed my comments here in my last 2 reviews :)
These internal properties are not needed right? Can we call isIpad() and isIphone() directly on line 211?

Reply:
I've added these back in order to unit test insightspublisher, so mostly for dependency injection.